packages/app/universal-ts-utils/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './array'
```

Comment:
do we need this file.

Reply:
Already removed, I think we can without it for now, if the main "index" begins to be too big we can consider splitting but for now I think we don't need it